src/runtime-rs/crates/hypervisor/src/dragonball/inner_device.rs

```diff
@@ -56,13 +56,16 @@ impl DragonballInner {
                     config.no_drop,
                 )
                 .context("add block device"),
-            Device::Vsock(config) => self.add_vsock(&config).context("add vsock"),
+            Device::HybridVsock(config) => self.add_hvsock(&config).context("add vsock"),
             Device::ShareFsDevice(config) => self
                 .add_share_fs_device(&config)
                 .context("add share fs device"),
             Device::ShareFsMount(config) => self
                 .add_share_fs_mount(&config)
                 .context("add share fs mount"),
+            Device::Vsock(_) => {
+                todo!()
```

Comment:
So the PR is still work-in-progress?

Reply:
This PR focuses on runtime-rs entirely. Dragonball hasn't support vhost-vsock yet. Our patch is based on our own vmm. Thanks to @Tim-0731-Hzt , this patch is verified by Qemu.

> Thanks @yipengyin , just verified using Qemu, it works